csrc/preprocess/cpu/image2tensor_impl.cpp

```diff
@@ -20,8 +20,9 @@ class ImageToTensorImpl : public ::mmdeploy::ImageToTensorImpl {
     int channels = shape[3];
 
     auto dst_mat = Transpose(Tensor2CVMat(src_tensor));
-
-    auto dst_tensor = CVMat2Tensor(dst_mat);
+    cv::Mat dst_mat_float;
```

Comment:
I think with the help of #208 it's no need to change the implementation of image2tensor. 
Beside,  the definition of `ImageToTensor` in [mmdet](https://github.com/open-mmlab/mmdetection/blob/6b87ac22b8d9dea8cc28b9ce84909e6c311e6268/mmdet/datasets/pipelines/formatting.py#L67) doesn't do data type cast